package.json

```diff
@@ -29,15 +29,15 @@
   "main": "lib/index",
   "readmeFilename": "README.md",
   "dependencies": {
-    "lodash": "3.10.x",
+    "lodash": "^4.17.10",
     "validator": "3.22.x"
   },
   "devDependencies": {
-    "restify": ">=2.8.1 <5.0.0",
-    "mocha": "3.1.x",
-    "sinon": "1.11.x",
+    "coveralls": "^3.0.2",
+    "mocha": "^5.2.0",
+    "restify": ">=2.8.1 <8.0.0",
```

Comment:
Unfortunately `node-restify-validation` doesn't actually work yet with restify 7. I'll need to sit down and try to work out what the problem is (you're welcome to give it a stab as well!).